src/views/TableDetail.vue

```diff
@@ -242,10 +242,12 @@ export default Vue.extend({
       });
 
       const {
-        rows,
+        results,
         count,
       } = result;
 
+      const rows = results;
+
```

Comment:
I *think* the following is a more concise way to pull out the `results` property and rename it `rows` in one shot. You can try this out locally and see if it works.

```suggestion
        results: rows,
        count,
      } = result;

```

Reply:
I tested it and it worked!